docs/Design/File Command.md

```diff
@@ -0,0 +1,138 @@
+# File
```

Comment:
Can you add a main description? What is a "File" in this context? It's not really clear if you're referring to a code file, a resource file, reading a file from disk, etc.